favorite_files.py

```diff
@@ -394,9 +425,18 @@ def check_st_version():
             window.run_command('open_file', {"file": "${packages}/FavoriteFiles/messages/upgrade-st-3080.md"})
 
 
+def settings():
+    return sublime.load_settings("favorite_files.sublime-settings")
+
+
 def plugin_loaded():
     """Setup plugin."""
+    global Favs, global_favs_file
 
-    global Favs
-    Favs = Favorites(join(sublime.packages_path(), 'User', 'favorite_files_list.json'))
+    global_favs_file = join(sublime.packages_path(), 'User', 'favorite_files_list.json')
+    Favs = Favorites(global_favs_file)
```

Comment:
I made a global for the global json file because it is used also in Toggle command